wit_dashboard/wit-dashboard.html

```diff
@@ -4892,18 +4993,18 @@ <h2>Create similarity feature</h2>
                       this.selectedSecondBreakdownFeature
                     );
                     facetedStats = inferenceStats.faceted[facetKey];
```

Comment:
why define facetedStats to be two different things l4995 l5007?

Reply:
whoops line 4995 was now incorrect but was being corrected by 5007. removing 4995.